spec/features/cluster_sizing_spec.rb

```diff
@@ -17,16 +17,10 @@
     let(:cloud_framework) { 'azure' }
     let(:instance_types) do
       Cloud::InstanceType.load(
-        Rails.root.join('config', 'data', "#{cloud_framework}-types.json")
+        Rails.root.join('vendor', 'data', "#{cloud_framework}-types.json")
```

Comment:
actually we are mis-abusing the vendor directory on all our fork. We should use config instead of vendor but ok for moment

Reply:
Yes. As our fork uses some specific data to create the cluster sizing page, I thought that the easiest was just to use the correct config file.

Reply:
anyways is not important in our context